content/blog/2022/Dates/MOTORSAW-AV-at-Vision'R-2022/index.md

```diff
@@ -0,0 +1,64 @@
+---
+categories: "Dates"
+author: "sunep"
+date: "2022-10-24"
+title: "MOTORSAW AV at Vision'R"
+description: "Deuterium Live AV performance"
+where: 
+    location_name: "Le Sample"
+    city: "18 Av. de la République, 93170 Bagnolet, France"
```

Comment:
Sune, would you please split the `city` into separate fields.

In general we have these fields for the `where`:

```
where: 
    location_name: ""
    street: ""
    additional: ""
    city: ""
    country: ""
    country_name: ""
    from_date: ""
    to_date: ""
    lat: ""
    long: ""
```

Thank you!